src/main/scala/cromwell/binding/command/Command.scala

```diff
@@ -54,7 +54,7 @@ object Command {
 case class Command(parts: Seq[CommandPart]) {
   val ws = Pattern.compile("[\\ \\t]+")
   def inputs: Seq[TaskInput] = {
-    parts.collect {case p: ParameterCommandPart => p}.map {p =>
+    val collectedInputs = parts.collect {case p: ParameterCommandPart => p}.map {p =>
```

Comment:
If I did a `parts.filter{_.isInstanceOf[ParameterCommandPart]}` the doesn't come out to `Seq[ParameterCommandPart]`.  Am I doing it wrong?